File: app/src/main/java/by/d1makrat/library_fm/ui/fragment/tabTop/TabTopAlbumsFragment.java
package by.d1makrat.library_fm.ui.fragment.tabTop;

import by.d1makrat.library_fm.adapter.pages.TabTopAlbumsAdapter;

public class TabTopAlbumsFragment extends TabTopItemsFragment<TabTopAlbumsAdapter> {

    @Override
    protected TabTopAlbumsAdapter createAdapter() {
        return new TabTopAlbumsAdapter(getChildFragmentManager());
    }
}

File: app/src/main/java/by/d1makrat/library_fm/ui/fragment/tabTop/TabTopAlbumsFragment.kt
package by.d1makrat.library_fm.ui.fragment.tabTop

import by.d1makrat.library_fm.adapter.pages.TabTopAlbumsAdapter

class TabTopAlbumsFragment : TabTopItemsFragment<TabTopAlbumsAdapter>() {

    override fun createAdapter(): TabTopAlbumsAdapter {
        return TabTopAlbumsAdapter(childFragmentManager)
    }
}
